_includes/buttons/add-to-calendar-button.html

```diff
@@ -16,7 +16,7 @@
   },
   data: {
     // Event title
-    title: '{{ title }}',
+    title: '{{ title | replace: "&", "and" }}',
```

Comment:
```suggestion
    title: '{{ title | url_encode }}',
```

Reply:
But I haven't tested this fully so give it a shot locally and see if the titles still look ok! There can be weirdness passing between places.

Reply:
Sadly I'm not sure that works. I just tried this locally and am getting "allies%2C Collaboration %26 Automation - ....", at least in Google Calendar. I'm not sure if I tried the url_encode filter, but I know others were returning similar unicode like output

Reply:
Ok one last thing to try - encodeURIComponent() in JavaScript

Reply:
Oh and it has to be url_encode the others aren’t intended for urls.

Reply:
okay I tested three solutions - all are no good! So this is our best bet for now. Reading more, it seems like most people use the calendar API proper for this kind of thing.

Reply:
Sorry I missed this last night. Yeah I tried the encodeURIComponent option too, but I'm glad you verified it didn't work right either. I was less familiar with the JS, so I thought maybe I didn't quite do it correctly. I noticed the button JS plugin we are using hasn't been updated since 2017, so maybe part of the true solution is finding a newer, maintained button that can do similar functionality. Or maybe it's just use the APIs proper ourselves.

Thanks for digging into this deeper!